console/src/test/java/org/georchestra/console/integration/UsersIT.java

```diff
@@ -176,14 +220,59 @@ private void setRole(String userName, String role1Name, String role2Name) throws
                 .andExpect(jsonPath("$.geodocs").value(3))//
                 .andExpect(jsonPath("$.ogcStats").value(3));
 
-        this.mockMvc.perform(post("/account/gdpr/delete?uid=user1"))//
+        this.mockMvc.perform(post("/account/gdpr/delete").header("sec-username", "user1"))//
+                .andExpect(status().isNotFound());
+
+    }
+
+    @WithMockRandomUidUser
+    @DBUnit(qualifiedTableNames = true, dataTypeFactoryClass = PostgresExtendedDataTypeFactory.class)
+    @DataSet(executeScriptsBefore = "dbunit/geonetwork_ddl.sql", strategy = CLEAN_INSERT, value = { "dbunit/all.csv" })
+    public @Test void testDeleteUserAsSuperAdminDoesNotDeleteGDPRAccountRecords() throws Exception {
+        createUser("user1");
+
+        deleteUser("user1")//
+                .andExpect(status().isOk());
+
+        Account account = new AccountImpl();
+        account.setUid("user1");
+        DeletedAccountSummary afterDeletion = gdprWorker.deleteAccountRecords(account);
+        assertFalse(afterDeletion.isEmpty());
+    }
+
+    @Ignore // don't know how to make it pass the checkAuthorization(String uid) check in
+            // UsersController
+    @WithMockRandomUidUser
+    @DBUnit(qualifiedTableNames = true, dataTypeFactoryClass = PostgresExtendedDataTypeFactory.class)
+    @DataSet(executeScriptsBefore = "dbunit/geonetwork_ddl.sql", strategy = CLEAN_INSERT, value = { "dbunit/all.csv" })
+    public @Test void testDeleteUserDeletesGDPRAccountRecords() throws Exception {
```

Comment:
cf. testDeleteAccountRecords, can be deleted ?

Reply:
done.